ext/enchant/enchant.c

```diff
@@ -483,6 +483,10 @@ PHP_FUNCTION(enchant_broker_get_dict_path)
 			RETURN_FALSE;
 	}
 
+	if (value == NULL) {
+		RETURN_FALSE;
```

Comment:
should this emit a warning/notice?

Reply:
I'm not sure what the common PHP practice is here, I'm don't like returning false that much since it doesn't give the impression the programmer did something wrong.

But I do believe it's a lot better then just segfaulting.